test/integration/stackstorm/controls/mongodb_test.rb

```diff
@@ -24,7 +25,7 @@
     # Mongo should listen on localhost only
     # @link: https://github.com/StackStorm/st2-packages/blob/a93701d98a130f50f7cb551e842889212ece3b11/scripts/st2bootstrap-deb.sh#L446-L447
     its(['net.bindIp']) { should eq '127.0.0.1' }
-    its(['net.port']) { should eq 27017 }
+    its(['net.port']) { should eq 27_017 }
```

Comment:
Any details behind this particular change?

Reply:
would you like me to try and disable this check?

Reply:
No, it's OK. Was just curious about this hack